hadoop-ozone/dev-support/checks/acceptance.sh

```diff
@@ -52,11 +52,6 @@ if [[ "${OZONE_ACCEPTANCE_SUITE}" == "s3a" ]]; then
   download_hadoop_aws "${HADOOP_AWS_DIR}"
 fi
 
-if [[ "${OZONE_ACCEPTANCE_TEST_TYPE}" == "robot" ]]; then
-  install_virtualenv
-  install_robot
```

Comment:
Are these already present on the docker images?

Reply:
Yes.  Robot tests are run in containers, so they must have Robot installed.

https://github.com/apache/ozone-docker-runner/blob/c3fe81f9cc94e4f0a429cf1bf5a7f441a3a1f701/Dockerfile#L61